linkerd/core/src/main/scala/io/buoyant/linkerd/FailureAccrualInitializer.scala

```diff
@@ -16,7 +16,7 @@ abstract class FailureAccrualConfig extends PolymorphicConfig {
   var backoff: Option[BackoffConfig] = None
 
   @JsonIgnore
-  val backoffOrDefault =
+  lazy val backoffOrDefault =
```

Comment:
Okay, if we're limited by Jackson in this case, I'll definitely make it a `def` for now. It would be nice to use a parsing library that generates immutable immutable config classes but that sounds like it would be a major structural change at this point.

I tend to make fairly liberal use of `lazy val`s in my own code, but this is because I generally try to make everything immutable – i've been known to set WartRemover to make `var` into a compile error. In this case, you're right, and I'll change it to a `def`.